etc/inc/allow-pandoc.inc

```diff
@@ -0,0 +1,9 @@
+# This file is overwritten during software install.
+# Persistent customizations should go in a .local file.
+include allow-pandoc.local
+
+noblacklist ${PATH}/pandoc
+noblacklist /usr/bin/pandoc
+noblacklist /usr/lib/ghc-*
+noblacklist /usr/lib/*pandoc*
+whitelist /usr/share/pandoc*
```

Comment:
Remove.

None of the `allow-*.inc` includes have `whitelist`ing options as they break profile which do not `whitelist`ing in this location (exception is `allow-opengl-game.inc` because all profiles using it have wusc+`private-bin`).